### PR TITLE
Added a workflow with GitHub action to check for CRLF line endings

### PR DESCRIPTION
**Pull request by PurpleBriar**
# Type of PR

- [ ] Documentation changes
- [ ] Code changes
- [ ] Test changes
- [x] CI-CD changes
- [ ] GitHub Template changes

## PR Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] My code follows the code style of this project.
- [ ] I ran lint checks locally prior to submission.
- [x] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Purpose of PR

Added a workflow to check for CRLF line endings on pull requests

## Does this introduce a breaking change

- [ ] YES
- [x] NO

## Validation

- [ ] Unit tests updated and ran successfully
- [x] Update documentation or issue referenced above

## Issues Closed or Referenced

- Closes retaildevcrews/wcnp/issues/875
- References #issue_number (this references the issue but does not close with PR)

### CODE REVIEW

File: .github/workflows/crlf-check.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  check_pull_request_job:
+    runs-on: ubuntu-latest
+    name: Check files for CRLF line endings
+    steps:
+    - name: Checkout repository contents
+      uses: actions/checkout@v1
+
+    - name: Validate files
+      run: |
+        chmod +x "./.github/crlf-check.sh"
```
Comment (AAkindele) [inline .github/workflows/crlf-check.yaml:17]: update the file to have the executable flag and commit that so this line can be removed.
```suggestion
```

`git update-index --chmod=+x path/to/file`
https://git-scm.com/docs/git-update-index#Documentation/git-update-index.txt---chmod-x

Comment (PurpleBriar) [reply .github/workflows/crlf-check.yaml]: Replaced custom code with github action

**Review by alfredoihernandez — DISMISSED** [review]
Why not detect mixed line endings instead of presence of CRLF?

File: .github/workflows/crlf-check.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+jobs:
+  check_pull_request_job:
+    runs-on: ubuntu-latest
+    name: Check files for CRLF line endings
+    steps:
+    - name: Checkout repository contents
+      uses: actions/checkout@v1
+
+    - name: Check for CRLF endings
+      uses: erclu/check-crlf@v1
```
Comment (alfredoihernandez) [inline .github/workflows/crlf-check.yaml:16]: For this repo, we talked about having this check occur during in build workflow

**Review by alfredoihernandez — APPROVED** [review]

**Pull request merged**